back-end/legeno-around-here/src/main/java/wooteco/team/ittabi/legenoaroundhere/dto/UserCreateRequest.java

```diff
@@ -0,0 +1,42 @@
+package wooteco.team.ittabi.legenoaroundhere.dto;
+
+import wooteco.team.ittabi.legenoaroundhere.domain.user.Email;
+import wooteco.team.ittabi.legenoaroundhere.domain.user.NickName;
+import wooteco.team.ittabi.legenoaroundhere.domain.user.Password;
+import wooteco.team.ittabi.legenoaroundhere.domain.user.User;
+
+public class UserCreateRequest {
+
+    private String email;
+    private String nickName;
+    private String password;
+
+    public UserCreateRequest() {
+    }
+
+    public UserCreateRequest(String email, String nickName, String password) {
+        this.email = email;
+        this.nickName = nickName;
+        this.password = password;
+    }
+
+    public User toUser() {
+        Email email = new Email(this.email);
+        NickName nickName = new NickName(this.nickName);
+        Password password = new Password(this.password);
+
+        return new User(email, nickName, password);
+    }
```

Comment:
바로 return 해줘도 괜찮을 것 같아요!
```suggestion
    public User toUser() {
        return new User(
            new Email(this.email),
            new NickName(this.nickName),
            new Password(this.password)
        );
    }
```

Reply:
반영했습니다!